coralproject/talk
# [CORL-879] Add an unanswered tab to stream when in Q&A mode

## What does this PR do?

Adds an Unanswered tab to the comment stream when the user converts the Stream to a Q&A mode. Unanswered top level comments (questions) are tagged as Unanswered.

## How do I test this PR?

- Convert a stream to Q&A using the config tab
- Head back to the stream (should now show Q&A on root tab)
- See that there is a new tab column called "Unanswered"
